react/game/src/app/screens/Game/index.js

```diff
@@ -111,4 +75,29 @@ class Game extends Component {
   }
 }
 
-export default Game;
+Game.propTypes = {
+  history: arrayOf(shape({})),
+  xIsNext: bool,
+  stepNumber: number,
+  winner: string,
+  playerMove: func.isRequired,
+  changeHistory: func.isRequired
+};
+
+const mapStateToProps = state => ({
+  history: state.game.history,
+  xIsNext: state.game.xIsNext,
+  stepNumber: state.game.stepNumber,
+  winner: state.game.winner
+});
+
+const mapDispatchToProps = dispatch => ({
+  playerMove: (history, winner, squares) =>
+    dispatch(actionCreators.handlePlayerMove(history, winner, squares)),
+  changeHistory: step => dispatch(actionCreators.handleHistoryChange(step))
+});
+
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(Game);
```

Comment:
Let's split this component in a smart and presentational component. All redux connection should be placed in container component, and int the presentational one u must only handle the layout.
Check [this](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0) and [this](https://medium.com/@learnreact/container-components-c0e67432e005)